bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqData.java

```diff
@@ -118,11 +119,8 @@ private String getGarageStatus(int attemps) {
 	private boolean Login() {
 		String url = String
 				.format("%s/api/user/validate?appId=%s&SecurityToken=null&username=%s&password=%s",
-						this.webSite, this.appId, this.userName, this.password);
```

Comment:
using "this" is not necessary with class scope, its mainly used when a method declares its own local var with the same name as a class local one